### PR TITLE
Accessbility changes

### PR DESCRIPTION
**Pull request by ayebaishmo**
In this branch I was able to write a report  on accessibility for the site, The following was changed.
Changed the title to give a description about the content.
Added alt text to images that never had.
Added labels to contact form for the screen only readers.
The rest had no issues.

### CODE REVIEW

**Review by Shaher-11 — APPROVED** [review]
Hello @ayebaishmo ,

# Congratulations! 🎉 
#### Your project is approved!  There is nothing else to say other than... it's time to merge it :muscle: :muscle: 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Cheers and Happy coding!👏👏👏

**Comment by eri8-9** [comment]
# Required Changes ♻️ ♻️ 

Hi @ayebaishmo, 

Great work  👏 
You've done well, but there are still some which aren't addressed yet.

## Aspects to improve (only one) ♻️
_Check the comments under the review._

- [ ] You need to have a report summary of the evaluation you have made on your PR. Each summary list should also be linked to the code snippet where file changes are made in the GitHub files. Please carefully read the project's requirements, make the necessary changes before you request another review.
- What you need to do here is just get better your summary in the PR. For each bullets, you need to have a link where file changes are made in the GitHub files. Do that and you'll be on the other side (the good one). Use [links](https://www.google.com/) like this.
- hint: `[links](https://www.google.com/)`


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -4,7 +4,7 @@
 <head>
   <meta charset="utf-8"/>
   <title>
-    portifolio
+    Ishmael's portifolio
```
Comment (eri8-9) [inline index.html:7]: The improvement of the title was great just one thing, "portifolio" is grammatically incorrect in the English language instead use "Portfolio"

**Review by eri8-9 — COMMENTED** [review]
Modify in your README file: "portifolio" is grammatically incorrect in the English language instead use "Portfolio"

File: index.html (hunk 1 above)
Comment (ayebaishmo) [reply index.html:7]: Thanks I wold change that

**Pull request merged**